R/utils.R

```diff
@@ -1,12 +1,10 @@
 convert_log2_scale <- function(
-  dt_in, vars_to_transform = "value",
-  simplify_limits = TRUE) {
+  dt_in,
+  lower_limit,
+  vars_to_transform = "value") {
   dt_out <- data.table::copy(dt_in)
   for (var in vars_to_transform) {
-    if (simplify_limits == TRUE) {
```

Comment:
At the moment am just warning if there are values outside of provided detection limits... but perhaps if there are, we should also keep this logic, i.e. set all higher values to the upper limit and all lower values to the lower limit?

Reply:
Yeah, so we are now doing this when the data is first passed in, iff the `strict_upper_limit`/`strict_lower_limit` args are TRUE.